CONTRIBUTING.md

```diff
@@ -85,13 +85,37 @@ pytest tests/unit
 
 ### Building & Testing Documentation
 
+#### Livereload w/automatic rebuild
```

Comment:
```suggestion
### Automatic Rebuild with Live Reload
```

Reply:
I made this change but kept the fourth-level header `####` unless you wanted to make them third-level headers instead?  Would have to also change the other one.